elastic/apm-server
# add ilm policiy to apm data streams

## Motivation/summary

This copies the current ILM policy to all data streams in the package. 
We should determine if we want to change it for some/all data streams. 
I'm happy to do that as part of this PR or as follow up.


## Checklist


For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

With apm-integration-testing:
`./scripts/compose.py start master --no-download --with-elastic-agent --no-apm-server --with-package-registry --package-registry-apm-path <path-to>/apm-server/apmpackage/apm` install the APM integration and check that the ILM policies for all data streams are installed.

## Related issues
